sdk/src/openssl/ec_validator.rs

```diff
@@ -198,28 +191,4 @@ mod tests {
         let validator = EcValidator::new("es256");
         assert!(!validator.validate(&signature, &data, &pub_key).unwrap());
     }
-
-    #[test]
```

Comment:
Is this no longer a valid / needed test?

Reply:
All the new certs are cert chains so this is now redundant.